test/test_workchains.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 import inspect
-from tornado import gen
 
 import plumpy
 from plumpy.workchains import *
 from plumpy.process_listener import ProcessListener
 import unittest
+import pytest
+import asyncio
```

Comment:
move import to top

Reply:
The reason is that all files under `test/` are excluded for `pylint`. We may want to change this, but not in this PR